bin/package-browser

```diff
@@ -4,6 +4,14 @@ ROOT=`dirname $0`
 
 cd "$ROOT/.."
 
+echo "Updating translations..."
```

Comment:
This approach has the nice side-effect of automatically bundling translations in our releases as well